src/Entity/ProductTrait.php

```diff
@@ -13,7 +13,7 @@ trait ProductTrait
      * @ORM\ManyToOne(targetEntity="Loevgaard\SyliusBrandPlugin\Entity\Brand", inversedBy="products")
      * @ORM\JoinColumn(name="brand_id", referencedColumnName="id")
      */
-    protected $brand;
+    private $brand;
```

Comment:
This breaks backwards combatibility. Can you revert this?

Reply:
i do that, because doctrine:migration:diff no see it in protected, but just in private. And i don't know bhy :(